image_publisher/package.xml

```diff
@@ -9,27 +9,28 @@
   </description>
   <author>Kei Okada</author>
   <maintainer email="vincent.rabaud@gmail.com">Vincent Rabaud</maintainer>
+  <maintainer email="chris.ye@intel.com">Chris Ye</maintainer>
   <license>BSD</license>
   <url>http://ros.org/wiki/image_publisher</url>
 
-  <buildtool_depend>catkin</buildtool_depend>
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <test_depend>ament_cmake_gtest</test_depend>
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
 
   <build_depend>cv_bridge</build_depend>
-  <build_depend>dynamic_reconfigure</build_depend>
-  <build_depend>camera_info_manager</build_depend>
+  <!--build_depend>camera_info_manager</build_depend-->
```

Comment:
camera_info_manager is available now (or once I get this last PR merged this morning)